.github/workflows/kicad-release.yml

```diff
@@ -0,0 +1,83 @@
+name: KiCad Release
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: "The version string for this release"
+        required: true
+        type: string
+      name:
+        description: "The name of the project"
+        required: true
+        type: string
+      dir:
+        description: "The directory where to project folder resides"
+        required: true
+        type: string
+
+jobs:
+  build-artifacts:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out project
+        uses: actions/checkout@v3
+        with:
+          lfs: true
+  
+      - name: Change version in file
+        run: |
+          sed -i '/rev /s/".*"/"'$VERSION'"/' $FILE.kicad_sch
```

Comment:
I find quiet dangerous to have a so broad search and replace. If something contains "rev" it is replaced.

I did some test with `sed -i 's/(rev "[0-9a-zA-Z._-]*")/(rev "'4.6.10'")/' foo.txt` and it was working on my machine.

I think also this command should fail if nothing is found but I don't know how to do it.

Reply:
Fixed the restricted parsing in fa2ef61.

For the "fail if nothing found": I don't think it's a problem: if nothing is found, nothing is changed, which means the revision has not been set in the document, which is bad practice but still acceptable. Also the generated artifacts will still have the proper version number.

Reply:
True. Failing has it pro and cons. When bad practice is wanted it is bad, when it is a mistake it is good (best way to detect something, better than warning).

Let keep it this way, it's not a big issue if the document is not labeled correctly (it is still in the artifacts).